utils.sh

```diff
@@ -3,20 +3,20 @@
 set -e
 
 verify_checksum() {
-  FILE=$1
-  EXPECTED_CHECKSUM=$2
-  FILE_CHECKSUM=$(sha256sum ${FILE} | cut -d ' ' -f 1)
+  FILE="$1"
+  EXPECTED_CHECKSUM="$2"
+  FILE_CHECKSUM="$(sha256sum "${FILE}" | cut -d ' ' -f 1)"
 
-  if [[ $FILE_CHECKSUM != $EXPECTED_CHECKSUM ]]; then
+  if [[ "$FILE_CHECKSUM" != "$EXPECTED_CHECKSUM" ]]; then
     echo "Checksum mismatch!"
     return 1
   fi
 }
 
 package() {
-  PKGNAME=$1
-  VERSION=$2
+  PKGNAME="$1"
+  VERSION="$2"
 
-  cd $PREFIX
-  tar --remove-files --create --gzip --file $PKGDIR/bootstrap-$PKGNAME-$VERSION.tar.gz .
+  tar -C "$PREFIX" -czvf "$PKGDIR/bootstrap-$PKGNAME-$VERSION.tar.gz" .
```

Comment:
Please keep long flags here thanks.

Reply:
Done in 6755c2cb723962e11458951b03c88c68f73ceadf.